### ISSUE TITLE
[1.12] Issues regarding ModTweaker and Aether Legacy

### ISSUE THREAD

**Issue by Spearkiller** [Modding-Legacy/Aether-Legacy#301]
So I've been messing around with CustomStuff4 and Mod Tweaker these past few days in order to add stuff to make certain mods harder, and I thought a good idea for that was to add stuff in the Aether, one of which was mob drops for each of the bosses, making it so going to the Aether and actually defeating enemies/bosses was essential to progress in some mods. With the help of a friend, we managed to get mob drops to work... for every mob but ones from the Aether.

I don't know how, or why, (or even if this is the place I should be saying this, rather than over on the Mod Tweaker page :x ) but when I try to add any Aether entity to the list of mobs to drop a certain item, i get this error: https://cdn.discordapp.com/attachments/435094864829480960/435254733935083520/unknown.png
(AAA Spear's Addition's Mob Drops is the name of the .zs file, where the drops were added.)
Using /mt entities also doesn't add Aether legacy entities into the log where the rest are displayed.

So, basically, is there any way you could code in support for Mod Tweaker, allowing entity names to actually be able to be read by it, but also add the ability to add recipes to things such as the Enchanter or freezer. (Potentially even the incubator, if that would work)

My apologies if this should be on the ModTweaker GitHub, rather than here, but my friend said that it should be on here since it could be to do with the way how the Aether creates entities in a way that Mod Tweaker isn't familiar with or something.

Thanks in advance :) 

EDIT: Okay, after further attempts, this seems to be an issue with Mod Tweaker, not the Aether, as it's not possible to add drops to some mobs from other mobs as well. My apologies.
